tests/test_dataset.py

```diff
@@ -0,0 +1,98 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+from __future__ import absolute_import, division, print_function
+
+import os
+import unittest
+
+import lsst.utils.tests
+from lsst.verify.ap.dataset import Dataset
+
+
+class DatasetTestSuite(lsst.utils.tests.TestCase):
+
+    def setUp(self):
+        self._testbed = Dataset('HiTS2015')
+
+    def test_init(self):
+        """Verify that if a Dataset object exists, the corresponding data are available.
+        """
+        try:
+            import lsst.verify.ap.hits2015  # noqa: F401
```

Comment:
@djreiss, I don't think this will work with what you did for DM-11117. Do you have any suggestions for how to make sure a dataset package has been set up correctly?